helpers/middleman/locale.rb

```diff
@@ -39,6 +39,13 @@ def is_valid_locale_id?(locale_id)
     return !!data.locales.find { |x| x[:id]&.downcase == locale_id&.downcase }
   end
 
+  def is_valid_locale_id_for_base_url?(locale_id, base_url)
+    return true if locale_id.nil?
+    return true unless locale_id == 'us'
+
+    !base_url.match?(/help.sharesight.com/)
+  end
+
```

Comment:
I'd suggest a comment…this logic doesn't exactly tell me "the us locale is not available on the helpsite".